taskt/Core/Automation/Commands/Word Commands/WordCreateApplicationCommand.cs

```diff
@@ -2,64 +2,75 @@
 using System.Collections.Generic;
 using System.Windows.Forms;
 using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
 using taskt.UI.CustomControls;
 using taskt.UI.Forms;
-using Word = Microsoft.Office.Interop.Word;
-using taskt.Core.Utilities.CommonUtilities;
+using Application = Microsoft.Office.Interop.Word.Application;
 
 namespace taskt.Core.Automation.Commands
 {
 
     [Serializable]
-    [Attributes.ClassAttributes.Group("Word Commands")]
-    [Attributes.ClassAttributes.Description("This command creates a Word Application.")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to launch a new instance of Word.")]
-    [Attributes.ClassAttributes.ImplementationDescription("This command implements Word Interop to achieve automation.")]
+    [Group("Word Commands")]
+    [Description("This command creates a Word Instance.")]
+    [UsesDescription("Use this command when you want to launch a new instance of Word.")]
+    [ImplementationDescription("This command implements Word Interop to achieve automation.")]
```

Comment:
Remove **UsesDescription** and **ImplementationDescription** attributes

Reply:
done